### PR TITLE
Feature/caes 1466 batch import

### PR DESCRIPTION
**Pull request by sa715**

### CODE REVIEW

**Review by beshkenadze — APPROVED** [review]

File: packages/containers/Import/Steps/ImportingStep/ImportingStep.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const TwoItemsWrapper = styled.div`
 `;
 
 const renderText = progress =>
-  progress < 1
+  progress < 0.99999
```
Comment (cu4xxi) [inline packages/containers/Import/Steps/ImportingStep/ImportingStep.js:41]: why such a number?

**Review by cu4xxi — COMMENTED** [review]
And need to be refactored after release/2.0.0 will be merged

File: packages/containers/Import/Steps/ImportingStep/ImportingStep.js (hunk 1 above)
Comment (sa715) [reply packages/containers/Import/Steps/ImportingStep/ImportingStep.js:41]: Sometimes prop `progress` has value like 0.9999999999999 when import is already finished

Comment (cu4xxi) [reply packages/containers/Import/Steps/ImportingStep/ImportingStep.js:41]: o_O
`0.99` like a 99% 
// nitpick

**Review by cu4xxi — APPROVED** [review]

**Pull request merged**